packages/test/test-version-utils/src/testApi.ts

```diff
@@ -67,6 +68,7 @@ const packageList = [
 	"@fluidframework/local-driver",
 	"@fluidframework/odsp-driver",
 	"@fluidframework/routerlicious-driver",
+	"@fluidframework/agent-scheduler",
```

Comment:
hey @scottn12 FYI--when changing the set of packages that test-version-utils loads for compat purposes, it's good practice to bump the revision constant defined in `versionUtils.ts` so that other devs with existing install trees of the legacy modules don't encounter issues at runtime. I encountered this just now when merging over this change: got an error since agent-scheduler wasn't installed to 1.3.7 due to the incremental install logic in this/that file.

Not a huge deal, just wanted to raise some awareness in this realm for the future :)